ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -417,11 +417,15 @@ public class BottomSheetController: UIViewController {
         stackView.axis = .vertical
         stackView.translatesAutoresizingMaskIntoConstraints = false
 
+        bottomSheetContentView.accessibilityElements = [resizingHandleView]
```

Comment:
not clear why we need to manually add it ourselves?

Reply:
if the dimmingView is shown then do we need to mannually inject it too?

Reply:
Added a comment which explains why we're doing this.

Reply:
>if the dimmingView is shown then do we need to mannually inject it too?
No, this only enforces ordering within the actual sheet view. Dimming view is outside of this and gains focus before any of these.

Reply:
i am sure you already tried it but https://developer.apple.com/documentation/objectivec/nsobject/1615143-shouldgroupaccessibilitychildren on the bottomsheetcontentview doesn't help? VO doing its own thing?

Reply:
oh I haven't tried that! Let me see...

Reply:
Tried `shouldGroupAccessibilityChildren`, doesn't fix this issue unfortunately.